### DISCUSSION TITLE
CasaOS

### DISCUSSION THREAD

**Discussion by usbtux** [tteck/Proxmox#967] [Help Needed] [answered]
What is the default ip port to connect to please?

Ive tried the 443 but I'm net getting any where.

Ive noticed that this seems common on some of the install  Interface info

**Comment by tteck** [comment, marked as answer]
Requires no port being defined, as shown in the instructions "**CasaOS Interface: IP**"

**Reply by usbtux** [reply]
I restarted the container and retried - it now works! SO sorry to waste your time.
